.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+build
```

Comment:
we don't have a top level build directory, and `node_modules` is always excluded. I don't think this does anything

Reply:
Oh, I didn't think about top level. I excluded top level `build` directory in `.eslintignore`. and remove `node_modules`